api/tests/routes/native/v1/features_toggle_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+
+pytestmark = pytest.mark.usefixtures("db_session")
+
+
+class FeaturesToggleTest:
+    def test_patch_success(self, client):
```

Comment:
Pour tester que la route modifie bien la valeur d'un feature flag, on peut forcer la valeur du flag en début de fonction avec `pcapi.core.testing.override_features()` : 

```suggestion
    @override_features(ENABLE_NATIVE_APP_RECAPTCHA=False)
    def test_patch_success(self, client):
```

...